web/src/features/fbaCalculator/components/windSpeedCell.test.tsx

```diff
@@ -0,0 +1,47 @@
+import { render, screen } from '@testing-library/react'
+import { FBAInputRow } from 'features/fbaCalculator/components/FBAInputGrid'
+import WindSpeedCell, {
+  WindSpeedCellProps
+} from 'features/fbaCalculator/components/WindSpeedCell'
+import React from 'react'
+describe('WindSpeedCell', () => {
+  const buildProps = (
+    inputRow: FBAInputRow,
+    calculatedValue?: number
+  ): WindSpeedCellProps => ({
+    fbaInputGridProps: {
+      inputRows: [inputRow],
+      stationOptions: [],
+      updateRow: () => {
+        /** no op */
+      }
+    },
+    classNameMap: { windSpeed: '' },
+    inputValue: inputRow.windSpeed,
+    calculatedValue,
+    rowId: 0
+  })
+  interface WindSpeedInput {
+    windSpeed: number | undefined
+    calculatedWindSpeed: number | undefined
+  }
+  const buildInputRow = (windSpeed: number | undefined) => ({
+    id: 0,
+    weatherStation: undefined,
+    fuelType: '',
+    grassCure: undefined,
+    windSpeed: windSpeed
+  })
+  it('should set input value if no calculated value', () => {
+    const row = buildInputRow(1)
+    const props = buildProps(row)
+    render(<WindSpeedCell {...props} />)
+    expect(screen.getByTestId('windSpeedInput-0').firstChild?.firstChild).toHaveValue(1)
+  })
+  it('should set calculated value if no input value', () => {
+    const row = buildInputRow(undefined)
+    const props = buildProps(row, 2)
+    render(<WindSpeedCell {...props} />)
+    expect(screen.getByTestId('windSpeedInput-0').firstChild?.firstChild).toHaveValue(2)
+  })
+})
```

Comment:
fun insanity testing: if I use the keyboard to type in a stupidly large wind speed value, everything blows up and there's no error message (e.g., https://wps-prod.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=322&f=o1b&c=100&w=11111111)

Quick bug fix for this PR, or something to add to the backlog for later? I'll leave it up to you to decide.

Reply:
Limited windspeed input to 100 both via stepper and keyboard.